tools/src/bin/web-server.rs

```diff
@@ -247,6 +247,15 @@ fn main() {
         res.send(&output).unwrap();
     };
 
+    {
+        let mut status_out = OpenOptions::new()
+            .append(true)
```

Comment:
Should this really append? I don't think it matters given the usage we're giving it but...

Reply:
Yeah the append is actually important because router.py writes to the same file and we don't want to clobber that one if it happens first. I'll add comments at both write locations.